### PR TITLE
[PORT] 9.0 account invoice shipping address

### PR DESCRIPTION
**Pull request by ghost**
This modules adds shipping address to invoice.

### CODE REVIEW

File: account_invoice_shipping_address/__openerp__.py [hunk 1]
```diff
@@ -23,9 +23,9 @@
 
 {
     "name": "Invoice Shipping Address",
-    'summary': """
+    'description': """
```
Comment (pedrobaeza) [inline account_invoice_shipping_address/__openerp__.py]: Don't put description. The description is in the README.rst file. This is only summary.

**Comment by pedrobaeza** [comment]
Thanks for the PR. I have seen that there's a very similar module on https://github.com/OCA/account-invoice-reporting/tree/9.0/account_invoice_delivery_address.

Can you tell me which one is better?

**Comment by ghost** [comment]
I think the module you mentioned is more complete than this. It controls the delivery address and also provides an RML report. What do you think of moving those to this PR and replacing it with QWeb report?

**Comment by pedrobaeza** [comment]
Yeah, that seems a better option.

File: account_invoice_shipping_address/__openerp__.py (hunk 1 above)
Comment (ghost) [inline account_invoice_shipping_address/__openerp__.py]: Just summary key?

Comment (pedrobaeza) [inline account_invoice_shipping_address/__openerp__.py]: From Odoo v8, the description is automatically binded from the README.rst file, so no need to duplicate it.

**Pull request closed without merging**